package.json

```diff
@@ -24,15 +24,28 @@
   },
   "license": "MIT",
   "dependencies": {
+    "aws-sdk": "^2.482.0",
     "body-parser": "^1.19.0",
     "cuid": "^2.1.6",
+    "dotenv": "^8.0.0",
     "express": "^4.16.4",
-    "express-graphql": "^0.8.0",
     "formidable": "^1.2.1",
-    "graphql": "^14.2.1"
+    "fs": "0.0.1-security"
```

Comment:
What's this for?

Reply:
I think I added it by mistake! Will remove